### PR TITLE
CHEAP-950 Add deleteVendorTester, addVendorTesterByEmailAddress mutations

### PR DESCRIPTION
**Pull request by junzhengca**

### CODE REVIEW

**Review by bholagabbar — APPROVED** [review]

File: src/app/controllers/VendorController.js [hunk 1]
```diff
@@ -2,26 +2,75 @@
...
+        this.update                        = this.update.bind(this);
+        this.createWithEmployee            = this.createWithEmployee.bind(this);
+        this.updateAllMenuItemsStatus      = this.updateAllMenuItemsStatus.bind(this);
     }
 
     // ADD MUTATION METHODS BELOW
 
+    /**
+     * Delete a vendor tester by ID.
+     * @param id Vendor tester's ID.
```
Comment (ralph-dev) [inline src/app/controllers/VendorController.js:22]: Isn't this supposed to be CustomerID to delete?

File: src/app/controllers/VendorController.js [hunk 2]
```diff
@@ -2,26 +2,75 @@
...
+            }).catch(e => {
+                reject(e);
+            });
+        });
+    }
+
+    /**
+     * Add a new vendor tester by email address.
+     * @param {string} id Vendor's ID.
+     * @param {string} email_address Email address to add as tester as.
```
Comment (ralph-dev) [inline src/app/controllers/VendorController.js]:
```suggestion
     * @param {string} email_address Customer's email address to add as a tester.
```

**Review by ralph-dev — APPROVED** [review]
LGTM

**Comment by junzhengca** [comment]
nope it is a seperate ID.

On Sep 22, 2019, at 11:07 AM, Ralph Maamari <notifications@github.com> wrote:

﻿

@ralph-dev commented on this pull request.

________________________________

In src/app/controllers/VendorController.js<https://github.com/cheapreats/js-sdk/pull/29#discussion_r326905770>:

>      }

     // ADD MUTATION METHODS BELOW

+    /**
+     * Delete a vendor tester by ID.
+     * @param id Vendor tester's ID.


Isn't this supposed to be CustomerID to delete?

**Pull request merged**